core/install.sh

```diff
@@ -191,6 +191,9 @@ add-user --role owner --password "${ADMIN_PASSWORD:-Nethesis,1234}" admin
 echo "Enable the events gateway for the node agent:"
 systemctl enable --now eventsgw@node
 
+echo "Restart systemd-journald:"
+systemctl restart systemd-journald
+
```

Comment:
If the change is just a workaround to a Debian issue, can we put in at line L33?

Reply:
I've found the error only on Debian, but maybe Ubuntu for CI is also affected.
And maybe even CS9 on other cloud platforms can be affected.

I do not see any drawback on restarting journald. Do you think we could face any problem?

Reply:
> Am I wrong?

The installation log is not lost. You can see it using `journalctl -m`.

The restart is need to see the logs form now on. If you do not restart it, promtail will not be able to see coming messages.

> I've another concern. If the install script runs during the machine first boot (i.e. cloud-init) is it safe to restart journald at that point?

Why it shouldn't be? By the way, for now installation is happening outside cloud-init